editoast/src/views/rolling_stocks/mod.rs

```diff
@@ -246,12 +278,16 @@ async fn update(
     let mut rolling_stock = data.into_rolling_stock_model(rolling_stock_id, version);
```

Comment:
The workflow of this function was acceptable before when we mixed the model and the changeset, but it's weird now. I'd suggest to refactor it into a transaction.

1. open a transaction
2. turn the form into a changeset
3. increase the version within the changeset
4. `update_or_fail`
5. validate
6. commit transaction